ex1b.sh

```diff
@@ -0,0 +1,35 @@
+#1
+ls - S
+
+#1a
+ls -S /etc/ | head -1 
+#Then I found the biggest file that is "services"
+
+#1b
+cat /etc/services
+
+#2
+ls -S -r
+
+#2a
+ls -S /etc/ | tail -1
+#Then I found the smallest file that is "motd"
+cat /etc/motd
+
+#3
+mkdir -p grandpa/father/son
+
+#4
+bc
+
+#5
+bc -q
+
+#6
+cal 5 2023

```

Comment:
They task is to write the year, this should be done dynamically, next year I want to run the project and see the year 2024, try to think how it can be done. try read about it here: https://www.tutorialkart.com/bash-shell-scripting/bash-current-year/